src/main/java/org/terasology/gooeyDefence/EnemyManager.java

```diff
@@ -155,54 +206,51 @@ public void update(float delta) {
      * @param entity the enemy to move
      */
     private void processEnemy(EntityRef entity, float delta) {
```

Comment:
I think if the name of the function was moveEnemyAlongPath, it wouldn't be necessary to read the javadoc to see what exactly the function does. Also if you at some point add more code to processEnemy doing something else, there is a danger of the javadoc becoming wrong.